src/components/GirlsHooHack/Footer/Footer.tsx

```diff
@@ -1,8 +1,36 @@
 import React from 'react';
-
+import blueWaves from '../../../assets/images/footerWaveGHH.png';
+import whiteLogo from '../../../assets/images/whitelogo.png';
+import email from '../../../assets/images/envelope-solid.svg';
+import instagram from '../../../assets/images/instagram-square-brands.svg';
```

Comment:
is the instagram logo supposed to have the square border around it?

Reply:
i think so